tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -2303,4 +2343,15 @@ public function data_envelope_params() {
 			array( array( 'alternate' ) ),
 		);
 	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @throws Exception
+	 *
+	 * @return array
+	 */
+	public function data_get_index_should_return_help_and_not_name() {
```

Comment:
The data provider should be placed directly below `test_get_index_fields_links()`.